src/alacris_lib/state_update.ml

```diff
@@ -6,75 +6,43 @@ open Types
 open Legilogic_ethereum
 open Operator_contract
 open Digesting
-open Ethereum_chain
+open Side_chain
 open Side_chain_operator
 open Side_chain_server_config
 
 let state_update_log = true
 
-let status_null_operation = ref 0
-
 let last_hash = ref Digest.zero
 let first_nontrivial_hash = ref Digest.zero
 
+let small_money_transfer : recipient:Address.t -> sender:Address.t -> unit Lwt_exn.t =
+  fun ~recipient ~sender ->
+  if state_update_log then
+    Logging.log "small_money_transfer_recipient_sender";
+  let open Lwt_exn in
+  let transfer_amount = TokenAmount.of_string "500000000000" in
```

Comment:
If we want toy numbers, then "1" would be enough.

If we want "realistic" numbers, then mind that 500 gwei is a small fraction of the typical cost of gas: 21000 gas * 1 gwei/gas (gas price typically varies from 1 to 10 gwei, more if you're playing games). If we go the "realistic" route, we should also include a comment with an explanation where the number comes from, e.g. "This .01 ETH of 1e16 wei is worth about $3 at the current July 2019 ~$300 USD/ETH rate, whereas a typical gas price for the transaction is worth 2.1e14 wei at 10 gwei/gas, or about 6 cents."

Reply:
Value 1 has been put.